web/packages/teleport/src/services/apps/types.ts

```diff
@@ -39,6 +39,12 @@ export interface App {
   samlApp: boolean;
   // samlAppSsoUrl is the URL that triggers IdP-initiated SSO for SAML Application;
   samlAppSsoUrl?: string;
+  // isGrafana is true if the name includes grafana or has the label "icon:grafana"
+  isGrafana?: boolean;
+  // isJenkins is true if the name includes jenkins or has the label "icon:jenkins"
+  isJenkins?: boolean;
+  // isSlack is true if the name includes slack or has the label "icon:slack"
+  isSlack?: boolean;
```

Comment:
this will probably continue to grow, i would do something like this:

`type GuessedAppType = 'grafana' | 'jenkins' | 'slack'`